python/lsst/ts/standardscripts/maintel/laser_tracker/align.py

```diff
@@ -0,0 +1,220 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["Align", "AlignComponent"]
+
+import enum
+
+import numpy as np
+import yaml
+from lsst.ts.idl.enums.LaserTracker import LaserStatus
+from lsst.ts.observatory.control import RemoteGroup
+from lsst.ts.observatory.control.maintel.mtcs import MTCS, MTCSUsages
+from lsst.ts.observatory.control.remote_group import Usages
+
+from lsst.ts import salobj
+
+
+# TODO - Remove this class and subsitute by class in idl.enum when available
```

Comment:
We need a ticket associated with this TODO. It can be the ticket that you will use to add it to ts-idl. In this case, make sure the ticket also mentions removing this here. 

Make sure you mention the ticket here, e.g.;

```
# TODO (DM-12345) - ...
```